docs/user/getting_started.rst

```diff
@@ -155,3 +149,6 @@ Both the RM3 and the OSWEC examples (``$WECSIM/examples/OSWEC``) come ready-to-r
     Storage <https://git-lfs.github.com/>`_ , or run 
     ``$WECSIM/examples/RM3/hydroData/bemio.m`` to generate the correct 
     ``rm3.h5`` file.
+
+.. Note:: You can remove the WEC-Sim Source folder from MATLAB path by running the "deactivate_wecsim.m" script in the ``$WECSIM folder. 
+However, it advisable to let the Source stay in MATLAB path.
```

Comment:
suggested grammatical change `However, it is advisable`

Reply:
@nathanmtom  sorry about that. Thanks for spotting it.